### ISSUE TITLE
Lowered letters in Hungarian text

### ISSUE THREAD

**Issue by pnorman** [versatiles-org/versatiles-style#29]
I'm seeing some letters in Hungarian text being lowered in the text line
![image](https://github.com/user-attachments/assets/19920d6f-ec24-43f2-9a75-9bf5124c5dbf)

You can see this at https://pnorman.github.io/tilekiln-shortbread-demo/#11.47/44.4357/26.1556 which is using the style provided in the releases

**Comment by hyperknot** [comment]
Hungarian speaker here, those are Romanian letters / cities. Still valid point though.

**Comment by Robot8A** [comment]
Yes, it seems to be happening with Romanian letters [Țț](https://en.wikipedia.org/wiki/%C8%9A) and [Șș](https://en.wikipedia.org/wiki/%C8%98), see attached image
![image](https://github.com/user-attachments/assets/fee319f2-4f15-4f7b-bde7-0af589f1b45b)

**Comment by hyperknot** [comment]
I'm quite sure it's the font, nothing to do with the style. I'm also in the process of making a correct Open Sans + Noto Sans font in MapLibre.
https://github.com/maplibre/maplibre-gl-js/issues/4820

**Comment by hyperknot** [comment]
I've posted my journey so far on the rabbit hole of open source maplibre fonts here:
https://github.com/maplibre/maplibre-gl-js/issues/4820#issuecomment-2478722628
